ts/components/services/ServiceList.tsx

```diff
@@ -15,38 +32,115 @@ type AnimatedProps = {
 };
 
 type OwnProps = {
+  sections: ReadonlyArray<SectionListData<pot.Pot<ServicePublic, Error>>>;
+  profile: ProfileState;
+  isRefreshing: boolean;
+  onRefresh: () => void;
+  onSelect: (service: ServicePublic) => void;
+  readServices: ReadStateByServicesId;
+  isExperimentalFeaturesEnabled?: boolean;
```

Comment:
we removed this prop, since at the momenti it is no more useful

Reply:
removed within https://github.com/teamdigitale/io-app/pull/1269/commits/fcd913a081c407763d30c57453e8320447843857